### PR TITLE
Clarify button message at each step of Upload

### PR DESCRIPTION
**Pull request by milesrichardson**
As part of adding the upload component to the dashboard, change the text of the button at each step of the flow.

I know that @onpaws is working on this page so I tried to keep edits minimal (rearrange and edit the assortment of renderings of text within the button).

image
image
image
image
image

Still todo:

- The first enabled CTA should say `"Upload N files"`

### CODE REVIEW

File: tdesign/src/Upload/Upload.tsx
Comment (Gelio) [inline tdesign/src/Upload/Upload.tsx:182]: A mighty number, looks like :smile:

File: tdesign/src/Upload/Upload.tsx
Comment (Gelio) [inline tdesign/src/Upload/Upload.tsx:195]: Not sure if that was intended to have the same button label in these 2 cases :thinking:

Comment (Gelio) [reply tdesign/src/Upload/Upload.tsx:195]: Nevermind, I see the "still TODO" in the PR description, all is good

**Review by Gelio — APPROVED** [review]
Looks alrighty to me :+1:

File: tdesign/src/Upload/Upload.tsx
Comment (milesrichardson) [reply tdesign/src/Upload/Upload.tsx:182]: lol this keeps happening because i use `cmd+2` to switch to the second pane - i even deleted one in this commit yesterday 😨

**Comment by onpaws** [comment]
These change look great btw

**Comment by milesrichardson** [comment]
Here's the messages now:


https://user-images.githubusercontent.com/835921/150620441-90bec60a-f27e-4ccb-acd7-aed472ad7eab.mp4



I also want to make the loading state at the end be different, but that might be a bigger change so I'm holding off until the refinements branch is merged.

**Review by Gelio — APPROVED** [review]
Still LGTM

**Comment by onpaws** [comment]
+1, still LGTM

**Comment by milesrichardson** [comment]
Thanks for the reviews – Merging this now

**Pull request merged**
